tarball/doc.go

```diff
@@ -39,7 +41,17 @@
 //		if err != nil {
 //			panic(err)
 //		}
-//		err = cp.Image(nil, dest, src, nil)
+//
+//		policy, err := signature.DefaultPolicy(&types.SystemContext{ /*...*/ })
```

Comment:
`DefaultPolicy(nil)` would be the simplest use, though the example given is of course valid.

Reply:
Changed.